src/client/components/ProductDetail/ProductDetail.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProductDetail.styles.css';
+import Button from '../Button/Button.component';
+
+export default function ProductDetail({
+  product,
+  quantityCount,
+  setQuantityCount,
+}) {
+  function decreamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount - 1);
+  }
+
+  function increamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount + 1);
+  }
+
+  function handleQuantityCountChange(e) {
+    e.preventDefault();
+    setQuantityCount(e.target.value);
+  }
+
+  const { name, price, picture } = product;
+  return (
+    <div className="productDetail">
+      <article className="productDetail-container">
+        <section className="image-container">
+          <img src={picture} alt={name} />
+        </section>
+        <section className="info-container">
+          <h3>{name}</h3>
+          <div className="info-container-stripes">
+            <div className="info-container-stripe">
+              <p>2lt pot </p>
```

Comment:
This feels like it should be some other product data, what more does the API return?

Reply:
I was just following the wireframe. At the moment we only got size and color as properties of a product.